admin/resources/js/components/Table.tsx

```diff
@@ -1,18 +1,33 @@
 /* eslint-disable react/jsx-key */
 import React from "react";
-import { useTable } from "react-table";
+import { useTable, useGlobalFilter, useSortBy, Column } from "react-table";
+import GlobalFilter from "./GlobalFilter";
 
 interface TableProps {
   columns: any;
   data: any;
 }
 
 const Table: React.FunctionComponent<TableProps> = ({ columns, data }) => {
-  const { getTableProps, getTableBodyProps, headerGroups, rows, prepareRow } =
-    useTable({ columns, data });
+  const {
+    getTableProps,
+    getTableBodyProps,
+    headerGroups,
+    rows,
+    prepareRow,
+    setGlobalFilter,
+    state,
+    visibleColumns,
+    preGlobalFilteredRows,
+  } = useTable({ columns, data }, useGlobalFilter, useSortBy);
 
   return (
     <table {...getTableProps()} style={{ border: "solid 1px blue" }}>
```

Comment:
I would recommend removing any inline css for now so that we are starting with a blank slate (at least stylistically).

Reply:
Sure. Will do.

Reply:
One more inline css to remove, `border: "solid 1px blue"`

Reply:
Wops. Removed.